README.md

```diff
@@ -220,10 +220,14 @@ Parameters:
   Custom plugins are supported via the `pluginLocation` parameter.
 * `mpsLocation` - optional location where to place the MPS files.
 * `javaExec` - optional `java` executable to use.
-* `plugins` - optional list of plugins to load before generation is attempted.
-  The notation is `new Plugin("someID", "somePath")`. Where the first parameter is the plugin id and the second the `short (folder) name`.
 * `pluginLocation` - location where to load the plugins from. Structure needs to be a flat folder structure similar to the
   `plugins` directory inside of the MPS installation.
+* `plugins` - optional list of plugins to load before generation is attempted.
+  The notation is `new Plugin("pluginID", "somePath")`. The first parameter is the plugin id.
+  For the second parameter `"somePath"` there are several options:
+  * if it's an absolute path, the plugin is loaded from this location
+  * if it's a folder located under `pluginLocation` the plugin is loaded from this location
+  * otherwise the plugin directory must be located under the default `mps/plugins` 
```

Comment:
otherwise _it should be a plugin folder_ located under...

Reply:
done